app/helpers/application_helper.rb

```diff
@@ -1,2 +1,18 @@
 module ApplicationHelper
+  def bootstrap_class_for flash_type
+    { success: "alert-success", error: "alert-danger", alert: "alert-warning", notice: "alert-info" }[flash_type.to_sym] || flash_type.to_s
+  end
+
+  def flash_messages(opts = {})
+    flash.each do |msg_type, message|
+      concat(content_tag(:div, message, class: "alert #{bootstrap_class_for(msg_type)} alert-dismissible", role: 'alert') do
+        concat(content_tag(:button, class: 'close', data: { dismiss: 'alert' }) do
+          concat content_tag(:span, '&times;'.html_safe, 'aria-hidden' => true)
```

Comment:
I can take this all out if it makes you guy uncomfortable because I'm not sure exactly what it's doing either. I thought it was really cool functionality to have a colored flash message with an x in the upper right corner to remove. I was looking into how to style flash messages with bootstrap and came upon that link I linked to in the commit message. If you'd like me to remove, just let me know!